src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs

```diff
@@ -312,6 +312,33 @@ public Animation LoadBvhAnimation(string bvhFilename, Vector3 scale = null)
             return ret;
         }
 
+        /// <summary>
+        /// Load facial animation and assign to model.
+        /// facialFilename is json format what we predefined.
```

Comment:
Could you use `<param>` tag to decribe method parameters?
```C#
<param name="facialFilename">The json format what we predefined. </param>
```

Reply:
Thank you. for more, I add some <param> for some missing APIs.